SS14.Changelog/Controllers/WebhookController.cs

```diff
@@ -23,7 +23,7 @@ public class WebhookController : Controller
         private static readonly Regex IsChangelogFileRegex = new Regex(@"^Resources/Changelog/Parts/.*\.yml$");
 
         private static readonly Regex ChangelogHeaderRegex =
-            new Regex(@"^\s*(?::cl:|🆑) *([a-z0-9_\- ]+)?\s*$", RegexOptions.IgnoreCase | RegexOptions.Multiline);
+            new Regex(@"^\s*(?::cl:|🆑) *([a-z0-9_\- ,]+)?\s*$", RegexOptions.IgnoreCase | RegexOptions.Multiline);
```

Comment:
```suggestion
            new Regex(@"^\s*(?::cl:|🆑) *([a-z0-9_\- ,&]+)?\s*$", RegexOptions.IgnoreCase | RegexOptions.Multiline);
```
Add ampersand too I guess